src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -65,7 +137,7 @@
                 "\n" +
                 "A separate CLP, ClusterCrosscheckMetrics, can cluster the results as a connected graph " +
                 "according to LOD greater than a threshold. ",
-        oneLineSummary = "Checks if all fingerprints appear to come from the same individual.",
+        oneLineSummary = "Program to check that all data in the input files appear to have come from the same individual.",
```

Comment:
Program to check that --> Checks that 

We use the word `Program` specifically to mean the jar.
Program = GATK, Picard
Tool = CrosscheckFingerprints

Reply:
hmmm. that isn't really consistent...we have "ProgramGroup" and we have the PROGRAM argument to CollectMultipleMetrics and we have @CommandLineProgramProperties....